test/export/test_experimental.py

```diff
@@ -0,0 +1,87 @@
+# Owner(s): ["module: dynamo"]
+# flake8: noqa
+import torch
+import torch._dynamo
+
+from torch._functorch.aot_autograd import aot_export_module
+from torch._higher_order_ops.strict_mode import strict_mode
+
+from torch.testing import FileCheck
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+def _mark_strict_DO_NOT_USE(cls):
```

Comment:
This looks like the user-facing API, right? Maybe we should expose it somewhere in export?

Reply:
Yes i will change it later.